wabt-compatibility.spec.ts

```diff
@@ -36,14 +35,26 @@ const INCOMPATIBLE_FILES = [
   "threads.0.wasm.out",
 ];
 
+// These two below arrays are used to select corresponding feature flags for corresponding files.
+const FEATURE_FILE_NAMES = [
```

Comment:
This seems rather complicated. How about this instead?

```javascript
const FEATURE_FLAGS_FOR_FILE = [
  'atomic.1.wasm.out': { 'threads': true }
];
```

or even a `Map`?

Reply:
My preference would be to always enable threads in wabt. We don't have feature flags for wasmparser, we only have one configuration so we should validate against a consistent configuration of wabt.

Reply:
> My preference would be to always enable threads in wabt. We don't have feature flags for wasmparser, we only have one configuration so we should validate against a consistent configuration of wabt.

I was thinking to wait until wabt turn this flag on by default but I'm not strongly on that :)

Reply:
> This seems rather complicated. How about this instead?
> 
> ```js
> const FEATURE_FLAGS_FOR_FILE = [
>   'atomic.1.wasm.out': { 'threads': true }
> ];
> ```
> 
> or even a `Map`?

Fixed 👍

Reply:
I actually like Eric's suggestion. Let's just always turn on threads.

Reply:
Will do in the next PR.